components/statistics/ChartBarBox/index.tsx

```diff
@@ -216,17 +220,28 @@ function ChartBarBox(props: PropsType) {
             <div className="w-full px-20  md:pr-30 md:pl-0 h-[230px] md:h-[475px]">
               {!loading && chartData?.length && (
                 <ParentSize>
-                  {({ width, height }) => (
-                    <BarChart
-                      type={type}
-                      width={width}
-                      height={height}
-                      data={chartData}
-                      days={currentDays}
-                      buckets={buckets}
-                      colorBlockchainMap={colorBlockchainMap}
-                    />
-                  )}
+                  {({ width, height }) => {
+                    const isMobile = width <= 640;
+
+                    const bottomAxisData = isMobile
+                      ? mobileBottomAxisData
+                      : DesktopBottomAxisData;
+                    const bottomAxis = bottomAxisData[currentDays];
+                    return (
+                      <BarChart
+                        width={width}
+                        height={height}
+                        data={chartData}
+                        buckets={buckets}
+                        isMobile={isMobile}
+                        bottomAxis={bottomAxis}
+                        colorBucketMap={colorBlockchainMap}
```

Comment:
Maybe it is better to pass a function instead of map
```suggestion
                        getBucketColor={(color) => colorBlockchainMap.get(bucket)}
```

Reply:
We need it since the tooltip in the Chart package also uses colorBucketMap.